tools/check/forbidden_strings_in_code.txt

```diff
@@ -164,7 +164,7 @@ Formatter\.formatShortFileSize===1
 # android\.text\.TextUtils
 
 ### This is not a rule, but a warning: the number of "enum class" has changed. For Json classes, it is mandatory that they have `@JsonClass(generateAdapter = false)`. If it is ok, change the value in file forbidden_strings_in_code.txt
-enum class===74
+enum class===76
```

Comment:
?

Reply:
It's Benoit who added a check to be sure we don't forget to set generateAdapter = false for enums we need to keep safe from proguard